### ISSUE TITLE
Converting npy files to TFRecords files

### ISSUE THREAD

**Issue by romass12** [SatelliteVu/SatelliteVu-AWS-Disaster-Response-Hackathon#4]
I have tried to convert .npy files to tfrecords using 
`tf.train.Feature(float_list=tf.train.FloatList(value=array))`
But, on reading features and labels from these TFRecords, it says expected string but got float32

**Comment by htxiao123** [comment]
> I have tried to convert .npy files to tfrecords using `tf.train.Feature(float_list=tf.train.FloatList(value=array))` But, on reading features and labels from these TFRecords, it says expected string but got float32

Hi. When I run the deep_learning, I get an error saying that the .tfrecords can't be found and I don't know what to do next. Are you running smoothly? I would be grateful if you could teach me.

**Comment by htxiao123** [comment]
> > I have tried to convert .npy files to tfrecords using `tf.train.Feature(float_list=tf.train.FloatList(value=array))` But, on reading features and labels from these TFRecords, it says expected string but got float32
> 
> Hi. When I run the deep_learning, I get an error saying that the .tfrecords can't be found and I don't know what to do next. Are you running smoothly? I would be grateful if you could teach me.

And I didn't go through classical_ml, data_quality, dataset_preparation these steps, I was running the deep_learning directly, I don't know if that would work,looking forward to your reply.
